tests/db/conftest.py

```diff
@@ -98,7 +98,7 @@ def fin():
 
 
 @pytest.fixture
-def inputs(user_vk, amount=1, b=None):
+def inputs(user_vk, amount=4, b=None, blocks=4):
```

Comment:
Just a note regarding the passing of arguments to a pytest fixtures. It is not really possible. At least not in the way that one would expect it to be with a normal function. 

There are at least two alternatives, both relying on the usage of the [`request`](http://docs.pytest.org/en/latest/builtin.html#_pytest.python.FixtureRequest) object:

* [Apply indirect on particular arguments](http://docs.pytest.org/en/latest/example/parametrize.html#apply-indirect-on-particular-arguments)
* [Fixtures can introspect the requesting test context](http://docs.pytest.org/en/latest/fixture.html#fixtures-can-introspect-the-requesting-test-context)



stackoverflow post: [py.test: Pass a parameter to a fixture function](http://stackoverflow.com/questions/18011902/py-test-pass-a-parameter-to-a-fixture-function)